src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -22,18 +22,18 @@ After you've completed setting up a project in Sentry, Sentry will give you a va
 
 You should `init` the Sentry Browser SDK as soon as possible during your page load:
 
-``` javascript
-Sentry.init({ dsn: 'https://<key>@sentry.io/<project>' });
-
+```javascript
 // When using npm, import Sentry
 import * as Sentry from '@sentry/browser';
+
+Sentry.init({ dsn: '___PUBLIC_DSN___' });
```

Comment:
Same here, please use this placeholder.
Probably just a copy paste mistake.

Reply:
Noted. Thanks!